.devcontainer/devcontainer.json

```diff
@@ -26,16 +29,39 @@
 				"GitHub.copilot",
 				"VisualStudioExptTeam.vscodeintellicode",
 				"ms-vscode.makefile-tools",
-				"ms-vscode.hexeditor",
 				"ms-python.python",
 				"donjayamanne.python-extension-pack",

```

Comment:
Are we actively using this extension

Reply:
no, that is why I removed it

Reply:
It is still there

Reply:
i was thinking of the wrong extension, it's removed now

Reply:
the python extension pack is useful to have IMO, might as well leave it

Reply:
Sounds good